test/companies/companies_test.exs

```diff
@@ -28,6 +36,31 @@ defmodule Companies.CompaniesTest do
     end
   end
 
+  describe "get!/1" do
+    test "retrieves by id" do
+      company = insert(:company)
+
+      company_id = company.id
+
+      assert %{id: ^company_id} = Companies.get!(company.id)
+    end
+
+    test "does not retrieve deleted record" do
+      company = insert(:company, is_deleted: true)
+
+      assert_raise(Ecto.NoResultsError, fn -> Companies.get!(company.id) end)
```

Comment:
Could you remove the parenthesis from this assert and the one on line 60? We aren't using them elsewhere so let's remain consistent 😁

Reply:
That sounds like a need for reconfiguring `mix format`! 😄